cmd/acra-server/acra-server.go

```diff
@@ -477,16 +478,17 @@ func main() {
 		logging.SetLogLevel(logging.LogDiscard)
 	}
 
+	ctx := context.TODO()
```

Comment:
Typically, `context.Background()` is used for top-level contexts established in **main** (or in other top-level contexts). `context.TODO()` is intended for method calls deeper in the stack, as a temporary placeholder until a usable context is threaded into the method.

Reply:
Yes. Thanks